docs/marketplace/jenkins/testing-distribution/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Setting Up Appcircle Testing Distribution Plugin
+sidebar_label: Testing Distribution
+description: Enhance powerful plugin to distribute your builds to appcircle
+tags:
+  [
+    testing-distribution,
+    ipa distribution,
+    apk distribution,
+    binary distribution,
+  ]
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+The Appcircle Testing Distribution plugin allows users to upload their apps and start distribution to test groups or individuals.
+
+### Install Appcircle Testing Distribution Plugin
+
+Go to your Jenkins dashboard and follow, Manage Jenkins > Plugins and search for "Appcircle Testing Distribution"
```

Comment:
Go to your Jenkins dashboard and navigate to Manage Jenkins > Manage Plugins. Then, search for "Appcircle Enterprise Store" in the available plugins section.